### PR TITLE
bugfix:TCC transaction Dubbo service nesting cannot open transaction

### PR DESCRIPTION
**Pull request by zhongfuhua**
### Ⅰ. Describe what this PR did
TCC transaction Dubbo service nesting cannot open transaction

### Ⅳ. Describe how to verify it
Use the Dubbo service and test the scenario as transaction - > Service 1 - > service 2

### CODE REVIEW

File: core/src/main/java/io/seata/core/context/RootContext.java [hunk 1]
```diff
@@ -58,7 +70,19 @@ public static void bind(String xid) {
...
-
+    
+    /**
+     * Bind type
+     *
+     * @param xidType
+     */
+    public static void bindType(String xidType) {
+        if (LOGGER.isDebugEnabled()) {
+            LOGGER.debug("bind type " + xidType);
```
Comment (xingfudeshi) [inline core/src/main/java/io/seata/core/context/RootContext.java]: LOGGER.debug("bind type:{}",xidType);

File: core/src/main/java/io/seata/core/context/RootContext.java [hunk 2]
```diff
@@ -84,7 +108,20 @@ public static String unbind() {
...
+    
+    /**
+     * Unbind temporary string
+     *
+     * @return the string
+     */
+    public static String unbindType() {
+        String xidType = CONTEXT_HOLDER.remove(KEY_XID_TYPE);
+        if (LOGGER.isDebugEnabled()) {
+            LOGGER.debug("unbind type " + xidType);
```
Comment (xingfudeshi) [inline core/src/main/java/io/seata/core/context/RootContext.java]: LOGGER.debug("unbind type:{}",xidType);

**Comment by xingfudeshi** [comment]
@zhongfuhua Could you provide a unit test to verify this?

**Comment by zhongfuhua** [comment]
No, but you can test it in seata-sample.




------------------ 原始邮件 ------------------
发件人: "xingfudeshi"<notifications@github.com>; 
发送时间: 2019年9月29日(星期天) 上午10:10
收件人: "seata/seata"<seata@noreply.github.com>; 
抄送: "1508397599"<1508397599@qq.com>; "Mention"<mention@noreply.github.com>; 
主题: Re: [seata/seata] TCC transaction Dubbo service nesting cannot opentransaction (#1684)




@zhongfuhua Could you provide a unit test to verify this?

**Comment by zhongfuhua** [comment]
Copy the unbind method, thank you for the correction!




------------------ 原始邮件 ------------------
发件人: "xingfudeshi"<notifications@github.com>; 
发送时间: 2019年9月29日(星期天) 上午10:05
收件人: "seata/seata"<seata@noreply.github.com>; 
抄送: "1508397599"<1508397599@qq.com>; "Author"<author@noreply.github.com>; 
主题: Re: [seata/seata] TCC transaction Dubbo service nesting cannot opentransaction (#1684)




@xingfudeshi commented on this pull request.
 
 
In core/src/main/java/io/seata/core/context/RootContext.java:
 > @@ -84,7 +108,20 @@ public static String unbind() {          }          return xid;      } - +     +    /** +     * Unbind temporary string +     * +     * @return the string +     */ +    public static String unbindType() { +        String xidType = CONTEXT_HOLDER.remove(KEY_XID_TYPE); +        if (LOGGER.isDebugEnabled()) { +            LOGGER.debug("unbind type " + xidType);  
LOGGER.debug("unbind type:{}",xidType);

**Comment by xingfudeshi** [comment]
@zhongfuhua hello?

**Pull request closed without merging**